src/Services/People.ts

```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
+import configTs from "../config";
+import { ICreatePerson, IUpdate } from "../Interfaces/Person";
+
+const url = `${configTs.URL}${configTs.PEOPLE_API}`;
+
+class PeopleServices {
+  static getAllPeople = async () => {
+    return await axios.get(`${url}/`);
+  };
+
+  static createPerson = async (personDetails: ICreatePerson) => {
+    return await axios.post(`${url}/`, personDetails);
+  };
+
+  static updatePerson = async (personDetails: IUpdate) => {
+    return await axios({
+      method: "patch",
+      url: `${url}/${personDetails.id}`,
+      data: {
+        name: personDetails.name,
+        favoriteColor: personDetails.favoriteColor,
+        favoriteAnimal: personDetails.favoriteAnimal,
+        favoriteFood: personDetails.favoriteFood,
+      },
+    });
+  };
```

Comment:
I'm not sure why you went for this Axios syntax here when you're using the shorthand syntax everywhere else.
Instead of reconstructing the `personDetails` object, why not just filter out the fields you're not interested in?